MANIFEST.in

```diff
@@ -3,6 +3,7 @@ graft compliance
 graft example
 graft docs
 graft test
+graft bench
```

Comment:
Do we want to ship the benchmark test as part of the pypi release?
/cc @pgjones

Reply:
My bias is no, but it looks like in general people do. 

I'm happy to merge this as is (with benchmarks in), given I don't have a good argument not to include it.